WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -1518,9 +1518,10 @@ private void setCurrentTag(final ReaderTag tag) {
                 mTagPreviewHistory.push(tag.getTagSlug());
                 break;
             case BLOG_PREVIEW:
-                // fallthrough
+                // noop
+                break;
```

Comment:
Thanks for adding the `break`! I'll admit that I refrained from raising this in fear it would sound too nit-picky but, I think it's the proper thing to do. Falling-through to a case that is not relevant on one hand, and having just a `no-op` comment on the other felt too much of an optimization :)